src/canvas.class.js

```diff
@@ -1138,18 +1147,71 @@
      * @param {Event} [e] Event (passed along when firing "object:selected")
      * @return {Boolean} true if the selection happened
      */
-    _setActiveObject: function(object, e) {
-      if (this._activeObject === object) {
+    _setActiveObject: function (object, e) {
+      var isCollection = Array.isArray(object._objects), activeObject = this._activeObject;
```

Comment:
For fixing the drag of multiple object isCollection should be fixed  and line 1151 should be

`var isCollection = (Array.isArray(object._objects) &&  object.type != 'activeSelection'), activeObject = this._activeObject;`

Reply:
Thanks! I'll look into it.
Just wanted to make sure you've tested this with #7529 merged into your local branch.

Reply:
I didn't test it but I have a few thoughts.
If I understand correctly if we accept your proposal selecting objects nested in active selection will become impossible.
Can you confirm?
If so this case should be handled by the dev overriding `onSelect` of active selection.

Reply:
for me its working as it should selecting in icollection when its there but doesnt cancel selection for active selection, tested on #7529 merged branch.